packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -123,7 +123,7 @@ const DeactivateAccountReason = ({ onBackClick, mt5_login_list, client_accounts,
     React.useEffect(() => {
         if (total_checkbox_checked === max_allowed_reasons) setIsCheckboxDisabled(true);
         else if (is_checkbox_disabled) setIsCheckboxDisabled(false);
-    }, [total_checkbox_checked]);
+    }, [is_checkbox_disabled, total_checkbox_checked]);
```

Comment:
We are adding `is_checkbox_disabled` as dependency and we are changing the same prop in this useEffect.
I think it can cause a cycle here.